taipy_gui_ext_library/frontend/webpack.config.js

```diff
@@ -23,11 +23,18 @@ module.exports = (env, options) => {
     mode: options.mode, // 'development' | 'production'
     entry: ["./src/index.ts"],
     output: {
+      // filename: the filename of the JavaScript bundle that gets generated.
+      // The 'get_scripts()' method of the ElementLibrary subclass must return
+      // the full path to this file.
       filename: "library.js",
       path: path.resolve(__dirname, "dist"),
       library: {
+        // name: must a Camel-case version of the extension library name
```

Comment:
must *be* a Camel...

Reply:
(the default implementation of get_js_name())

Reply:
As long as your `get_js_name()` really means `get_js_module_name()`, I'm good with that.